src/Routes/ImageManager/steps/imageOutput.js

```diff
@@ -24,19 +27,20 @@ export default {
         value: release,
         label: releaseMapper[release],
       })),
-      initialValue: 'rhel-8.3',
+      initialValue: 'rhel-8',
       validate: [{ type: validatorTypes.REQUIRED }],
       isRequired: true,
       isDisabled: true,
     },
     {
       component: componentTypes.SELECT,
-      name: 'outputType',
+      name: 'imageType',
       label: 'Output Type',
-      options: [
-        { value: 'rhel-edge-iso', label: 'RHEL for Edge Installer (.iso)' },
-      ],
-      initialValue: 'rhel-edge-iso',
+      options: Object.keys(imageTypeMapper).map((imageType) => ({
+        value: imageType,
+        label: imageTypeMapper[imageType],
+      })),
```

Comment:
Since you are already mapping trough the keys, it might be better to use `entries`
```suggestion
      options: Object.entries(imageTypeMapper).map(([imageType, imageTypeTitle]) => ({
        value: imageType,
        label: imageTypeTitle,
      })),
```